packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue

```diff
@@ -277,4 +326,11 @@ onMounted(() => {
 	color: var(--color-warning);
 	line-height: 1.2;
 }
+.droppable {
+	border: 1.5px dashed var(--color-ndv-droppable-parameter) !important;
+}
+.activeDrop {
+	border: 1.5px solid var(--color-success) !important;
```

Comment:
can this be done without !important ?

Reply:
no, we need to override defaults of N8nInput(ElInput)